tests/run-make/forced-unwind-terminate-pof/foo.rs

```diff
@@ -0,0 +1,17 @@
+// Tests that forced unwind through POF Rust frames wouldn't trigger our terminating guards.
```

Comment:
It would be nice to have a Windows version of this test that uses `longjmp`. On Windows `longjmp` is implemented using SEH exceptions and effectively acts like a forced unwind.

Reply:
We already have a test longjmp-across-rust added by #48572 to fix #48251